### ISSUE TITLE
Create Components that can do block breaks / block places.

### ISSUE THREAD

**Issue by rourke750** [gmlaxfanatic/Contraptions#28]
Possible feature is having certain factories that have pipes that go to other blocks and break either a block or row of blocks in front of it.

**Comment by gmlaxfanatic** [comment]
A major issue here is that block changes trigger chunk loading, so these
would either be chunk loaders themselves or require chunk loaders in one
form or another.

A fix could be caching the breaks, but that will have interesting
intrachunk edge cases.

**Comment by rourke750** [comment]
We could make it so the chunk had to be loaded and implement chunk loaders
as well.

**Comment by gmlaxfanatic** [comment]
Yeah, but I think encouraging chunk loading is the opposite direction we
want to head performance wise. A core role of RealisticBiome is to remove
the need for chunk loading.

Also, we already have chunk loaders in game, basically just as
players/bots.
